release_guidelines.md

```diff
@@ -8,73 +8,96 @@ We follow [semver](http://semver.org/) for versioning. Given a version number MA
+    - latest `<VERSION>` release should be installed
+10. Create a new release on GitHub, using release tag:
+    - title should be a version number `<VERSION>`
+    - as description use the entries for that version from CHANGELOG.md
+11. Merge `release/<release_version>` PR to `master`
+12. Merge `master` to `development`
+    * `git checkout master && git pull -p`
+    * `git checkout develop && git pull -p`
+    * `git merge master`
+    * `git push`
+13. After the release: [Update Sample App](#update-sample-app)
 
 ## Deploying the release to S3 production
 Deploying `dist/` folder to S3 is a crucial part of the release. It allows us to have a working demo of the SDK in JSFiddle and, more importantly, to support code splitting and lazy loading when the SDK is imported using HTML.
 
-* Make sure version is bumped in `package.json`
-* Make sure the `dist/` folder is updated and commited (by `npm run build`)
-* Run `aws s3 sync ./dist s3://onfido-assets-production/web-sdk-releases/<VERSION> --exclude "*.html" --exclude "*.map" --acl public-read --delete`
-* Make sure `dist/style.css`, `dist/onfido.min.js` and `dist/onfido.crossDevice.min.js` are in the S3 folder
+- Make sure version is bumped in `package.json`
+- Make sure the `dist/` folder is updated and commited (by `npm run build`)
+- Run `aws s3 sync ./dist s3://onfido-assets-production/web-sdk-releases/<VERSION> --exclude "*.html" --exclude "*.map" --acl public-read --delete`
+- Make sure `dist/style.css`, `dist/onfido.min.js` and `dist/onfido.crossDevice.min.js` are in the S3 folder
 
 Now you can go on and update JSFiddle.
 
 ## Update JSFiddle Demo
-* Make sure [Deploying the release to S3 production](#deploying-the-release-to-S3-production) step is executed before
-* See if these exist:
-  * `https://raw.githubusercontent.com/onfido/onfido-sdk-ui/<tag>/dist/style.css`
-  * `https://raw.githubusercontent.com/onfido/onfido-sdk-ui/<tag>/dist/onfido.min.js`
-* If they exist, copy each of them and paste on https://rawgit.com
-* Open the JSFiddle and update its resources to the following:
-  * `https://cdn.rawgit.com/onfido/onfido-sdk-ui/<tag>/dist/style.css`
-  * `https://cdn.rawgit.com/onfido/onfido-sdk-ui/<tag>/dist/onfido.min.js`
-* Follow the migration notes and update the code if necessary
-* Test the happy path
+- Make sure [Deploying the release to S3 production](#deploying-the-release-to-S3-production) step has been executed before
+- See if these exist:
+  - `https://raw.githubusercontent.com/onfido/onfido-sdk-ui/<tag>/dist/style.css`
+  - `https://raw.githubusercontent.com/onfido/onfido-sdk-ui/<tag>/dist/onfido.min.js`
+- If they exist, copy each of them and paste to https://rawgit.com
+- Open the JSFiddle and update its resources to the following:
+  - `https://cdn.rawgit.com/onfido/onfido-sdk-ui/<tag>/dist/style.css`
+  - `https://cdn.rawgit.com/onfido/onfido-sdk-ui/<tag>/dist/onfido.min.js`
+- Follow the migration notes and update the code if necessary
+- Test the happy path
```

Comment:
There should be another point here:
`- Copy the new JSFiddle link into README.md`

Reply:
done!